c2corg_api/models/userdata.py

```diff
@@ -0,0 +1,58 @@
+import bcrypt
+
+from sqlalchemy import (
+    Column,
+    Integer,
+    String
+    )
+
+from c2corg_api.models import Base, users_schema
+
+
+class UserData(Base):
+    """
+    Class containing the users' private and authentication data.
+    """
+    __tablename__ = 'userdata'
+    __table_args__ = {"schema": users_schema}
+    __mapper_args__ = {
+        'polymorphic_identity': 'ud'
```

Comment:
Not sure. But in c2cgeoportal we use it even for models with no inheritance:
https://github.com/camptocamp/c2cgeoportal/blob/master/c2cgeoportal/models.py#L196